### ISSUE TITLE
"reuse" of document without creating links between source document and new document

### ISSUE THREAD

**Issue by Goonie1984** [jesper-raemaekers/python-polarion#99]
Hello
I'm using "document.reuse" to create a copy of an existing document. By default the documents and contained work items from source to copy are linked with 'derived_from'. 
Is there a way to use "reuse" without creating the links resp. to get a standalone copy without any links to the source document?
I would be grateful for any ideas.

Thanks!

**Comment by jesper-raemaekers** [comment]
Hi,

Currently that is not possible. However I tried a way to do this. If you take these changes locally: https://github.com/jesper-raemaekers/python-polarion/commit/931982e502167a2a55e260ab23e9c4947a450f3d

You should then be able to create a copy of the document without links like this:

```
doc.reuse('target_project', 'target_space', 'new id' , 'new title', None)
```

This change will find its way to a release, but I cannot say when yet.

**Comment by jesper-raemaekers** [comment]
This update was released in V1.2.0 and I'm now closing this issue. Please reopen it if this did not resolve your question.

**Comment by Goonie1984** [comment]
Apologies for my late reply. Many thanks for extending the functionality - works like a charm!
